client/src/CodeChatEditor.mts

```diff
@@ -33,10 +33,20 @@ import "./graphviz-webcomponent-setup.mts";
 import "./graphviz-webcomponent/index.min.mjs";
 import { html_beautify } from "js-beautify";
 import { tinymce, tinymce_init } from "./tinymce-webpack.mjs";
+import TurndownService from "turndown";
```

Comment:
Aack, is the default import the only way to get to this? It's a [really bad thing](https://esbuild.github.io/content-types/#default-interop).

Reply:
From what I see, the only import available for this is the default

Reply:
Sigh. I'm not surprised.